src/resources/dicts/pt-br.edn

```diff
@@ -1,655 +1,673 @@
+ :help/context-menu "Bloquear menu de contexto"
  :help/markdown-syntax "Sintaxe de Markdown"
- :help/org-mode-syntax "Sintaxe de modo Org"
+ :help/org-mode-syntax "Sintaxe de Org mode"
  :bold "Negrito"
  :italics "Itálico"
- :highlight "Realçado"
- :strikethrough "Rasurado"
+ :highlight "Destaque"
+ :strikethrough "Tachado"
  :code "Código"
+ :untitled "Sem título"
  :right-side-bar/help "Ajuda"
- :right-side-bar/switch-theme "Temas"
+ :right-side-bar/switch-theme "Modos temáticos"
  :right-side-bar/contents "Conteúdo"
- :right-side-bar/page-graph "Grafo da página"
- :right-side-bar/block-ref "Referência de bloco"
+ :right-side-bar/page-graph "Página de grafo"
+ :right-side-bar/history "(Dev) Histórico de desfazer/refazer"
+ :right-side-bar/history-undos "Desfazer"
+ :right-side-bar/history-redos "Refazer"
+ :right-side-bar/history-pageonly "somente página"
+ :right-side-bar/block-ref "Referências de bloco"
+ :right-side-bar/graph-view "Visualização de grafo"
+ :right-side-bar/all-pages "Todas as páginas"
+ :right-side-bar/flashcards "Flashcards"
  :right-side-bar/new-page "Nova página"
+ :right-side-bar/show-journals "Mostrar diários"
+ :right-side-bar/separator "Manipulador de redimensionamento da barra lateral direita"
+ :right-side-bar/toggle-right-sidebar "Alternar barra lateral direita"
+ :right-side-bar/pane-close "Fechar"
+ :right-side-bar/pane-close-others "Fechar outros"
+ :right-side-bar/pane-close-all "Fechar todos"
+ :right-side-bar/pane-collapse "Recolher"
+ :right-side-bar/pane-collapse-others "Recolher outros"
+ :right-side-bar/pane-collapse-all "Recolher todos"
+ :right-side-bar/pane-expand "Expandir"
+ :right-side-bar/pane-expand-all "Expandir todos"
+ :right-side-bar/pane-open-as-page "Abrir como página"
+ :right-side-bar/pane-more "Mais"
+ :left-side-bar/switch "Troque para:"
  :left-side-bar/journals "Diários"
- :page/delete-confirmation "Tem certeza que quer apagar esta página e o arquivo associado?"
- :page/open-in-finder "Abrir em pasta"
- :page/open-with-default-app "Abrir com a aplicação por omissão"
- :page/make-public "Tornar pública para publicação"
+ :left-side-bar/create "Criar"
+ :left-side-bar/new-page "Nova página"
+ :left-side-bar/new-whiteboard "Novo whiteboard"
+ :left-side-bar/nav-favorites "Favoritos"
+ :left-side-bar/nav-recent-pages "Recentes"
+ :page/something-went-wrong "Algo deu errado"
+ :page/logseq-is-having-a-problem "Logseq está com um problema. Para tentar fazê-lo voltar a funcionar, siga as etapas seguras a seguir:"
+ :page/step "Etapa {1}"
+ :page/try "Tentar"
+ :page/slide-view "Visualizar como slides"
+ :page/slide-view-tip-go-fullscreen (fn [] [[:span.opacity-70 "Dica: pressione "] [:code "f"] [:span.opacity-70 " para ir para tela cheia"]])
+ :page/delete-confirmation "Tem certeza de que deseja excluir esta página e seu arquivo?"
+ :page/open-in-finder "Abrir no diretório"
+ :page/open-with-default-app "Abrir com o aplicativo padrão"
+ :page/make-public "Torná-la público para publicação"
```

Comment:
Oh, sorry, I don't think I gave enough details about this adjustment. When exchanging "Torná-lo" for "Torná-la" we must also exchange "público" for "pública" so that there is a verbal agreement.


I appreciate the change, and if you could change this more, that would be perfect!

Reply:
Imagine, I made the correction.